config/ci.json

```
@@ -12,7 +12,7 @@
     },
     "qa": {
         "default": {
-            "markers": "access_om3 or config"
+            "markers": "access_om3" # or config"
```

I don't think comments like this are allowed in json?
```suggestion
            "markers": "access_om3"
```